README.md

```diff
@@ -1,90 +1,105 @@
+
+## Description
+        
+  This modification allows you to control the air conditioner connected to ioBroker using acpartner.v3 (KTBL11LM), (it will probably work with version v2 too, but I don’t have a device to check, if anyone tries, let me know).
+  
+  When developing this adapter, the iobroker.mihome adapter version 1.2.9 was used, so I express my deep gratitude to all the authors who participated in this work: https://github.com/ioBroker/ioBroker.mihome/graphs/contributors.
+  
+  The main point is that apparently the Chinese clouds Xiaomi and Aqara use different commands in the network protocol. There is some documentation for Aqara - http://docs.opencloud.aqara.com/en/development/gateway-LAN-communication/#air-conditioning-controller. For Xiaomi, such documentation is not widely available. By connecting the acpartner gateway to the Aqara cloud, I received a firmware update that allowed the use of the air conditioning control commands described in the link above. A side effect is that there seems to be no miio protocol in Aqara firmware, at least acpartner is no longer visible to the miio-discovery command.
+  
+  The process of enabling LAN access and receiving GATEWAY KEY can be of some difficulty, the process is described below.
+  
+  To start using:
+  - Install the Aqara Home application on your smartphone (https://play.google.com/store/apps/details?id=com.lumiunited.aqarahome),
+  - register in the Aqara Home application,
+  - select the "Mainland China" region in the settings,
+  - add acpartner to the Aqara Home app,
+  - update the acpartner firmware (click on the air conditioning icon, then the three dots in the upper right corner, then click the lowest point “Software Version”), as a result, Aqara firmware will be installed on acpartner (when using the MiHome application it was from Xiaomi),
+  - register on the site https://opencloud.aqara.cn/ with the same password and login as in the Aqara Home application (registration confirmation may take some time, I had about 6 hours),
+  - log in to the console https://opencloud.aqara.cn/console/
+  - create an application on the tab https://opencloud.aqara.cn/console/app-management with the type "Device access" (I’m not sure about the need for this item (because I did it yet), so you can try to skip it),
+  - then go to the console https://opencloud.aqara.cn/console and select Gateway LAN on the left, fill in the "Aqara account" and "Password" fields and click the Submit button - you will see your Air Conditioning Controller and the network protocol enable button by clicking to which you allow LAN access and you will see the network key, which is necessary to configure the adapter in ioBroker.
+  - install the adapter from this page, in the settings enter the key obtained above.
+  
+  The following files have been modified with respect to the official version of the adapter:
+  - main.js
+  - hub.js
+  - Gateway.js
+  - devices.js
+  - THSensor.js
+  
+  
+## Описание
```

Comment:
Please have only english in README. A readme in other language can be created in a "docs/ru/README.md" folder and then linked.